website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -110,17 +110,34 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
  
-### SQLFluff integration
-The dbt Cloud CLI supports [SQLFluff](https://sqlfluff.com/), a modular and configuration SQL linter, which warns you of complex functions, syntax, formatting, and compilation errors.
+### Lint SQL files 
 
-To get started, run `dbt sqlfluff -h` to see the list of supported commands and flags, such as `dbt sqlfluff lint` to lint SQL files.
+From the dbt Cloud CLI, you can invoke [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. Many of the same flags that you can pass to SQLFluff are available from the dbt Cloud CLI, except those that might pose a security risk.
```

Comment:
I don't think we should include that line about a security risk. That's going to open us up to potential panicked support calls or confusion. The sentence also reads a little weird, I know this isn't the intent but the quick reader might look at that as us saying that using SQLFluff flags with dbt Cloud CLI is a security risk. Other reader might seem like we're calling using SQLFluff outside of our product a security risk, which could read as a slight. We could enhance the sentence and give specifics of what might be a risk, but I suggest removing the "security risk" portion entirely.
```suggestion
From the dbt Cloud CLI, you can invoke [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. Many of the same flags that you can pass to SQLFluff are available from the dbt Cloud CLI.
```